<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>971</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="new_dipendente">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aggiungi Dipendente</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>258</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ricerca Dipendenti</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_dipendente"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ruolo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combobox_ruoli"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>153</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkbox_attivo">
       <property name="text">
        <string>Attivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_inattivo">
       <property name="text">
        <string>Inattivo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="aggiorna_dipendente">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aggiorna Dipendente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_progetti">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Progetti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_crea_user">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Crea User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_edit_user">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_logout">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ordinamento</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combobox_ordinamento"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_azzera_filtri">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Azzera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
